### DISCUSSION TITLE
Modal over drawer does not work with me ?

### DISCUSSION THREAD

**Discussion by omanidev20** [saadeghi/daisyui#1165] [Q&A] [answered]
Hi All,
I have this issue since 3 days trying to find solution. I tried to play with z- index but no use. anyone has idea how can I make modal to be on top of sidebar

      <div class="modal z-50">
    <div class="drawer-side z-10">

<img width="905" alt="Screen Shot 2022-09-21 at 6 23 46 AM" src="https://user-images.githubusercontent.com/113412210/191400185-02be758b-58a3-48a7-b92e-3c8d9cc230b2.png">

**Comment by saadeghi** [comment, marked as answer]
Check out [modal examples](https://daisyui.com/components/modal/)  
and make sure you put the second section, before `</body>` tag.

![1](https://user-images.githubusercontent.com/7342023/191484634-e2f5273a-af64-442c-86ab-b29a577e6a82.png)

**Reply by omanidev20** [reply]
the model is inside drawer content . so I guess it is inside body tags.. I dont think this is the issue.

**Reply by RenovZ** [reply]
I still have a question, if I just cant put this before </body> tag, but I still want the modal overlay cover on the drawer-side, so how to do this?

**Reply by saadeghi** [reply]
If you're using [the checkbox method](https://daisyui.com/components/modal/#method-2-using-a-hidden-checkbox-legacy) for modal, you should put the specified part before `</body>` tag.

If you use [the new `<dialog>` element](https://daisyui.com/components/modal/#method-1-using-dialog-element-recommended), you can put it anywhere.
